app/Console/Commands/ImportSignupsCommand.php

```diff
@@ -52,6 +52,10 @@ public function handle()
         $signups_csv->setHeaderOffset(0);
         $missing_signups = $signups_csv->getRecords();
 
+        // Progress Bar
+        $number_of_missing_signups = count($signups_csv);
+        $bar = $this->output->createProgressBar(count($signups_csv));
```

Comment:
Want to use that `$number_of_missing_signups` var here?

Reply:
ah I meant to kill that one. Since getting the total count ended up not being messy and is only used once, I don't think it needs to live in its own variable